standards/request-response.md

```diff
@@ -471,6 +471,28 @@ X-SPS-User: 123456789012345678901234567890
 The usage of non-standard headers is not considered custom headers. For example, your API may consume or interact with other infrastructure outside of the contract specification for your service, such as `X-Forwarded-Host` or `X-Request-ID`, these are appropriate to continue using, but would not expect to identify them in your Open API spec for example. As such these types of headers not relevant to REST are intentionally left out of this documentation.
 ```
 
+#### SPS-CORS-Error
+
+**Type**: Response
+
+**Support**: OPTIONAL
+
+**Description**: A [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) error with an invalid request on a non-200 status code can be difficult to understand the reasons why the request was rejected. Using a custom header provides the ability to include more detailed information to the consumer on why their CORS preflight OPTIONS request was rejected.
```

Comment:
```suggestion
**Description**: A [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) error from an invalid request will return a non-200 status code, but it can be difficult for an API consumer to understand the reason that their request was unsuccessful. A custom header provides more detailed information to the consumer on why their CORS preflight OPTIONS request was rejected.
```